src/fdb5/api/SelectFDB.h

```diff
@@ -83,11 +92,10 @@ class SelectFDB : public FDBBase {
 
 private:  // members
 
-    std::vector<std::pair<SelectMap, FDB>> subFdbs_;
+    std::vector<FDBLane> subFdbs_;
```

Comment:
Since the code converges on using the term 'lane' I would also rename this field to `lanes_`